github3/issues/issue.py

```diff
@@ -240,6 +242,15 @@ def is_closed(self):
             return True
         return False
 
+    def is_locked(self):
+        """Checks if the issue is locked.
+
+        :returns: bool
+        """
+        if self.locked:
```

Comment:
Fixed in https://github.com/adrianmoisey/github3.py/commit/a600c31aee778824db8131075f30cb7aa3fb8690, thanks